packages/app/src/components/nominationBanner/style.css

```diff
@@ -76,3 +76,116 @@
 .hippa-column {
   white-space: nowrap;
 }
+
+
+/* modal style */
+
+.modal-background{
+  opacity: 1;
+  pointer-events: none;
+  transition: opacity 0.4s case-in-out ;
+  position: absolute; 
+  
+  z-index: 1000;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, 0.75);
+}
+
+
+/* .modalShowing-true {
+  opacity: 1;
+  pointer-events: visible;
+} */
+
+.modal-container{
+  z-index: 999;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  max-height: 90%;
+  overflow: inherit;
+  background: #fff;
+  box-sizing: border-box;
+  padding: 20px;
+  box-shadow: 0 1px 5px rgba(0,0,0,0.7);
+  border-radius: 4px;
+  width: 469px;
+  height: 250px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  align-content: flex-start;
+  display: flex;
+  pointer-events: visible;
+}
+
+.modal-text {
+  padding: 40px 40px;
+  font-weight: 300;
+  letter-spacing: .01em;
+
+}
+
+.modal-buttons {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  line-height: 20px;
+  padding-bottom: 19px;
+}
+
+
+.button-yes{
+  background-color: #128b3e;
+  color: white;
+  border: #999;
+  font-size: 15px;
+  font-weight: bold;
+  padding: 10px 40px 10px 40px;
+  margin-right: 10px;
+  height: 35px;
+  border-radius: 4rem;
+  font-size: 1rem;
+  letter-spacing: .01em;
+  line-height: 1rem;
+
+}
+
+.button-no{
+  background-color: #f72314;
+  color: #ffffff;
+  border: #8a8a8a;
+  font-size: 15px;
+  padding: 10px 40px 10px 40px;
+  margin-left: 10px ;
+  height: 35px;
+  border-radius: 4rem;
+  cursor: pointer;
+  font-size: 1rem;
+  letter-spacing: .01em;
+  line-height: 1rem;
+}
```

Comment:
There are a number of identical configuration here (color, height, border radius, letter spacing, line height); maybe it'd be worth having a shared class with some of those? Any particular reason we're using pixels (for, say, font-size) in one button but rems in the other?

Reply:
I hope I found a good way to share style for both buttons. There is no particular reason for using rem and/or pixels, it was just a mistake, sorry:)

Reply:
Looks good to me, thanks!